### PR TITLE
Update Jitsi to stable-6726-1

### PR DESCRIPTION
**Pull request by sakkiii**
Ref: https://github.com/jitsi/docker-jitsi-meet/releases/tag/stable-6726-1

### CODE REVIEW

File: roles/matrix-jitsi/templates/web/matrix-jitsi-web.service.j2 [hunk 1]
```diff
@@ -23,6 +23,7 @@ ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-jitsi-web \
 			{% endif %}
 			--mount type=bind,src={{ matrix_jitsi_web_config_path }},dst=/config \
 			--mount type=bind,src={{ matrix_jitsi_web_transcripts_path }},dst=/usr/share/jitsi-meet/transcripts \
+			--mount type=bind,src={{ matrix_jitsi_web_crontabs_path }},dst=/var/spool/cron/crontabs \
```
Comment (spantaleev) [inline roles/matrix-jitsi/templates/web/matrix-jitsi-web.service.j2:26]: What creates this crontabs file? Perhaps something is missing from `tasks/setup_install.yml`?

Comment (sakkiii) [reply roles/matrix-jitsi/templates/web/matrix-jitsi-web.service.j2:26]: I think this is for lets-encrypt renew certs crontab which will be manged by it's docker itself.

Comment (spantaleev) [reply roles/matrix-jitsi/templates/web/matrix-jitsi-web.service.j2:26]: As far as I know, you can't use `--mount` with a non-existing file (`/matrix/jitsi/web/crontabs` in this case).

Comment (sakkiii) [reply roles/matrix-jitsi/templates/web/matrix-jitsi-web.service.j2:26]: yes, mount directory should be an valid path. Thanks 👍

**Comment by spantaleev** [comment]
Thanks! :+1:

**Comment by spantaleev** [comment]
Possibly fixed in 60592fd6a8c3

**Pull request merged**